recipes/wgetter/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "wgetter" %}
+{% set version = "0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b6a4bef1a095e354d1dd24232947f5ee6cd5c6a63e0e34aeb9d7575620c371ee
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - wgetter
+ 
+about:
+  home: https://github.com/phoemur/wgetter
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Added PR ( https://github.com/phoemur/wgetter/pull/4 ) upstream to get the license file packaged in the PyPI `sdist`.